sources/packages/backend/apps/api/src/services/application/application.service.ts

```diff
@@ -1204,10 +1204,14 @@ export class ApplicationService extends RecordDataModelService<Application> {
         "programYear.startDate",
         "user.userName",
         "student.id",
+        "assessment.id",
```

Comment:
We don't need the "assessment.id" here.

Reply:
Sorry @andrepestana-aot, we actually need "assessment.id" because we are doing ".innerJoin("application.currentAssessment", "assessment")". The program will run into an error if we don't have this "assessment.id".

Reply:
Thanks for spotting this. It's probably a bug which I'm not sure, but the program will run into an error if we remove the "assessment.id". We can look into this further later on, and I think we can keep "assessment.id" for now.

Reply:
Got it. Thanks for explaining.